CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # Liana daemon and GUI release notes
 
+## 8.0
+
+This release introduces support for Bitbox02 device for a Taproot descriptor and
+a new install process including set-up templates and examples.
+The way the wallet displays its syncing state was reworked.
+
+### Breaking changes
+
+Running Liana v8 on an existing installation will migrate its database.
+Once migrated the database won't be compatible with previous versions of Liana.
+This means you won't be able to open with Liana v7 (or below) any wallet opened or created with Liana v8.
+
+The new Minimum Supported Rust Version of the GUI software is now 1.71.1. This change was necessary to include
+the new Bitbox02 rust client version.
+
+### Features
+
+#### Liana daemon / library
+
+A new field `last_poll_timestamp` was added to the wallet table in order to track the last time the wallet
+was updated by the poller.
+
+#### Liana GUI
+
+- The new Bitcoind 28.0 version is now downloaded by the GUI.
+- Bitbox02 devices can now be part of a Taproot descriptor set-up if their firmware version is superior or equal to 9.21.0.
+- The install process includes now example of setups with fixed templates and explanatory description to guide the user.
```

Comment:
"The install process now includes examples of setups with fixed templates and explanatory descriptions to guide the user."